src/main/resources/application.properties

```diff
@@ -93,3 +93,12 @@ sb.search.service.host=http://search-service:9000
 sb.composite.v4.search=/v4/search
 playlist_course_facets=courseCategory,primaryCategory
 elastic.required.field.designation.json.path=/EsFieldsmapping/designationEsRequiredFile.json
+
+knowledge.mv.service=http://knowledge-mw-service:5000/
+odcs.term.create=v1/framework/term/create
+odcs.framework.name=
+odcs.category.name=
```

Comment:
are these used only for designation ?? is it designation master / adding designation to org ?